File: C++/First Steps in Coding - Basics of C++/Conditional Statements - Advanced/Working Hours.cpp
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;
int main() {
	int hour;
	string day;
	cin >> hour; cin >> day;
	if (hour >= 10 and hour < 18) {
		if (day == "Monday" or day == "Tuesday" or day == "Wednesday" or day == "Thursday" or day == "Friday" or day == "Saturday") {
			cout << "open";
		}
		else {
			cout << "closed";
		}
	}
	else {
		cout << "closed";
	}
}

File: Java/Advanced Conditional Statements/Lab/07. Working Hours/WorkingHours.java
import java.util.Scanner;

public class WorkingHours {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int time = Integer.parseInt(input.nextLine());
        String day = input.nextLine();
        boolean open = false;

        switch(day){
            case "Monday":
            case "Tuesday":
            case "Wednesday":
            case "Thursday":
            case "Friday":
            case "Saturday":
                if(time >= 10 && time <= 18){
                    open = true;
                }
                break;
        }

        if(open){
            System.out.println("open");
        }else {
            System.out.println("closed");
        }
    }
}
